src/lib/types/option.ts

```diff
@@ -109,6 +110,7 @@ export interface IUnleashOptions {
     session?: Partial<ISessionOption>;
     getLogger?: LogProvider;
     logLevel?: LogLevel;
+    logLayout?: string | Layout;
```

Comment:
This change ends up exposing which internal logging framework Unleash uses, which should be an implementation  detail, and exposed to the end users. By exposing it we cannot change it without doing a major release.